main.cpp

```diff
@@ -108,8 +108,58 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Foot
+{
+    void stepForward();
+    int stepSize(int speed);
+};
 
-
+int stepSize(int speed)
+{
+    //float maxStep = 1.5f;
+    //float minStep = 0.3f;
+    
+    if (speed <=3)
+    {
+        return 3;
+    }
+    else if (speed >= 15)
+    {
+        return 15;
+    }
+    else
+    {
+        return speed;
+    }
```

Comment:
If you find yourself writing code like this:
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    else 
    {
        return anotherValue;
    }
}
```
if the `if()` block is entered, the `return aValue;` is the last line of code to be executed.
you have no code after the `else` block that needs executing, since your `else` block contains a `return` statement.
`return` statements are the last thing that **ever happens** in a function
That means if your `if()` block is **NOT** entered, it will be skipped.  What is the next thing that happens?  `return anotherValue;`
that means you do **NOT** need the else block, and it is just causing visual clutter.
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    return anotherValue;
}
```
- [ ] revise